app/templates/view_supplier_services.html

```diff
@@ -25,39 +25,53 @@
   {% with heading = "Services" %}
     {% include "toolkit/page-heading.html" %}
   {% endwith %}
-  {% call(item) summary.list_table(
-    services,
-    caption="Services",
-    empty_message="This supplier has no services on the Digital Marketplace",
-    field_headings=[
-      'Name',
-      'Service ID',
-      'Framework',
-      'Lot',
-      'Status',
-      summary.hidden_field_heading("Action")
-    ],
-    field_headings_visible=True
-  ) %}
-    {% call summary.row() %}
-      {{ summary.service_link(item.serviceName, '/' + item.frameworkFramework + '/services/' + item.id|string) }}
-      {{ summary.text(item.id) }}
-      {{ summary.text(item.frameworkName) }}
-      {{ summary.text(item.lotName) }}
-      {% call summary.field() %}
-        <span class="service-status-{{ item.status }}">
-          {% if item.status == "published" %}
-            Public
-          {% elif item.status == "enabled" %}
-            Private
-          {% elif item.status == "disabled" %}
-            Removed
-          {% else %}
-            {{ item.status|title }}
-          {% endif %}
-        </span>
+  {% set framework_services_grouper = dict(services | groupby('frameworkSlug')) %}
+  {% if not framework_services_grouper %}
```

Comment:
Can we simplify this if we're never going to show any tables? `{% if not services %}` just show "Services" header and the empty message?

Reply:
If you mean not pass the field headings etc. then yes (done!), if you mean change the design of the error message then I'd consider that out of scope. 
We should have though about this in story writing 😬 
Never mind. I'll have a look at the Trello ticket template and put a reminder on there or something. *Consider empty state*.
If design want a new type of error message on this page then that can be a new story.

Reply:
Done